app/models/service_template_ansible_playbook.rb

```diff
@@ -134,26 +144,55 @@ def validate_update_config_info(options)
     self.class.send(:validate_config_info, opts)
   end
 
-  def job_template(action)
+  def job_template!(action)
```

Comment:
Can you double check existing code where `job_template` was called? Do they need to change to `job_template!`?